.markdownlint.yaml

```diff
@@ -0,0 +1,9 @@
+line_length:              # MD013 max line length
```

Comment:
Since this is yaml, put a comment header at the top explaining what this is and how it's used?

Reply:
No problem. Worth mentioning that the existing yaml doesn't have one either: https://github.com/GoogleCloudPlatform/functions-framework-dart/blob/main/tool/markdown_linter_config.yaml

Reply:
Done.